package.json

```diff
@@ -87,26 +87,26 @@
     "release:major": "standard-version -a --release-as major"
   },
   "devDependencies": {
-    "@commitlint/cli": "17.2.0",
-    "@commitlint/config-conventional": "17.2.0",
+    "@commitlint/cli": "17.4.0",
+    "@commitlint/config-conventional": "17.4.0",
     "@cypress/code-coverage": "3.10.0",
     "@cypress/instrument-cra": "1.4.0",
-    "@trivago/prettier-plugin-sort-imports": "3.4.0",
+    "@trivago/prettier-plugin-sort-imports": "4.0.0",
```

Comment:
I don't expect this major update to break stuff, but just to make sure, do the prettier commands still work?

Reply:
[The `v4.0.0` changelog](https://github.com/trivago/prettier-plugin-sort-imports/releases/tag/v4.0.0) does not seem to mention breaking changes